Content.Shared/Doors/Systems/SharedAirlockSystem.cs

```diff
@@ -22,11 +24,46 @@ public override void Initialize()
 
         SubscribeLocalEvent<AirlockComponent, BeforeDoorClosedEvent>(OnBeforeDoorClosed);
         SubscribeLocalEvent<AirlockComponent, DoorStateChangedEvent>(OnStateChanged);
-        SubscribeLocalEvent<AirlockComponent, DoorBoltsChangedEvent>(OnBoltsChanged);
+        SubscribeLocalEvent<AirlockComponent, DoorBoltChangedEvent>(OnBoltsChanged);
         SubscribeLocalEvent<AirlockComponent, BeforeDoorOpenedEvent>(OnBeforeDoorOpened);
         SubscribeLocalEvent<AirlockComponent, BeforeDoorDeniedEvent>(OnBeforeDoorDenied);
         SubscribeLocalEvent<AirlockComponent, GetPryTimeModifierEvent>(OnGetPryMod);
         SubscribeLocalEvent<AirlockComponent, BeforePryEvent>(OnBeforePry);
+
+        SubscribeLocalEvent<AirlockComponent, ActivateInWorldEvent>(OnAirlockActivate);
+        SubscribeLocalEvent<AirlockComponent, InteractUsingEvent>(OnAirlockInteractUsing);
+    }
+
+    private void OnAirlockActivate(Entity<AirlockComponent> ent, ref ActivateInWorldEvent args)
+    {
+        if (args.Handled || !args.Complex)
+            return;
+
+        if (ent.Comp.KeepOpenIfClicked && ent.Comp.AutoClose)
+        {
+            ent.Comp.AutoClose = false;
+            Dirty(ent);
+        }
+    }
+
+    private void OnAirlockInteractUsing(Entity<AirlockComponent> ent, ref InteractUsingEvent args)
+    {
+        if (args.Handled || !_wiresSystem.CanWireAct(args.Used))
+            return;
+
+        args.Handled = true;
```

Comment:
This early Handled makes multitools no longer able to device link with doors, even those with closed wire panels. 

Move it down to inside the `panel.Open` if statement.